src/backend/commands/copyto.c

```diff
@@ -2167,7 +2167,14 @@ CopyToDirectoryTable(CopyToState cstate)
 						(errcode(ERRCODE_IO_ERROR),
 						 errmsg("Copy directory table to file failed, as file content is not consistent.")));
 
-		UFileClose(file);
+		int ret = UFileClose(file);
```

Comment:
`if (UFileClose(file) < 0)`

`ret` only used in if. and check others?

Reply:
done